File: Homework_1/Homework_1_C_Sharp/Homework_1_C_Sharp/Form1.cs
using System.Windows.Forms;
using System.Text;

namespace Homework_1_C_Sharp
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        
        

        private void progressBar1_Click(object sender, EventArgs e)
        {
            if (this.progressBar1.Value != 100)
            {
                this.progressBar1.Value += 1;
            }
            if (this.progressBar1.Value == 20)
            {
                this.BackColor = Color.Red;
            }
            if (this.progressBar1.Value == 40)
            {
                this.BackColor = Color.Yellow;
            }
            if (this.progressBar1.Value == 60)
            {
                this.BackColor = Color.Blue;
            }
            if (this.progressBar1.Value == 80)
            {
                this.BackColor = Color.Violet;
            }
            if (this.progressBar1.Value == 100)
            {
                this.BackColor = Color.RosyBrown;
            }

        }

 



        private void button1_Click(object sender, EventArgs e)
        {
            StringBuilder stringB = new StringBuilder();
            foreach (var Item in listBox1.Items)
            {
                stringB.Append(Item.ToString());
                stringB.Append(" ");

            }
            MessageBox.Show(listBox1.SelectedItem.ToString());
        }


    }
}

File: Homework_1/Homework_1_VB/Homework_1_VB/Form1.vb
Imports System.Text


Public Class Form1
    Private Sub Label1_Click(sender As Object, e As EventArgs) Handles Label1.Click

    End Sub

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Dim builder As New System.Text.StringBuilder
        For Each item As String In ListBox1.Items
            builder.Append(item.ToString())
            builder.Append(" ")

        Next
        MessageBox.Show(ListBox1.SelectedItem.ToString())

    End Sub


    Private Sub ProgressBar1_Click(sender As Object, e As EventArgs) Handles ProgressBar1.Click
        If Me.ProgressBar1.Value <> 100 Then
            Me.ProgressBar1.Value += 1
        End If
        If Me.ProgressBar1.Value = 20 Then
            Me.BackColor = Color.Red
        End If
        If Me.ProgressBar1.Value = 40 Then
            Me.BackColor = Color.Yellow
        End If
        If Me.ProgressBar1.Value = 60 Then
            Me.BackColor = Color.Blue
        End If
        If Me.ProgressBar1.Value = 80 Then
            Me.BackColor = Color.Brown
        End If
        If Me.ProgressBar1.Value = 100 Then
            Me.BackColor = Color.Wheat
        End If

    End Sub
End Class
